zbus_macros/src/proxy.rs

```diff
@@ -209,6 +209,9 @@ pub fn create_proxy(
             let property = method_attrs.property.as_ref();
 
             let rust_method_name = m.sig.ident.to_string();
+            let r#_stripped_rust_method_name = rust_method_name
```

Comment:
I think we can just shadow `rust_method_name` here instead of using a new name (just like for the function arguments). So this can just be:

```rust
let rust_method_name = rust_method_name
````

Reply:
More generally, we need both because there are two cases: We need the original when the identifier is reused as-is, like in proxy methods, because without `r#` it might not be valid there. And we need it without any `r#` when it is used as part of a derived identifier like `receive_{ident}` (in signals) or `cached_{ident}` (in properties), because there it's not valid _with_ `r#`.